docs/INSTALLING.md

```diff
@@ -0,0 +1,71 @@
+# How do I install the Code Corps Ember app?
+
+### Requirements
+
+You will need to install the following: 
+
+- [Node.js/npm](https://docs.npmjs.com/getting-started/installing-node) we use the LTS version (v 4.4.7)
```

Comment:
Is this versioning true? I believe we specify `4.5.0` which is what `ember-cli` specifies. I would also say we really need to update `package.json` to make the node engines more explicit.